pkg/asset/installconfig/platformpermscheck.go

```diff
@@ -58,7 +59,20 @@ func (a *PlatformPermsCheck) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return errors.Wrap(err, "validate AWS credentials")
 		}
-	case azure.Name, baremetal.Name, gcp.Name, libvirt.Name, none.Name, openstack.Name, ovirt.Name, vsphere.Name:
+	case gcp.Name:
+		client, err := gcpconfig.NewClient(context.TODO())
+		if err != nil {
+			return err
+		}
+		services, err := client.GetEnabledServices(ctx, "projects/"+ic.Config.GCP.ProjectID)
```

Comment:
It was good that you figured out the API issue here.

A few things:
1. concatenating strings within a function's parameters is hard to read. you should move this a variable, probably named `parent` 
2. Also you could probably do that string formatting within GetEnabledServices, so it is still just taking the `project` ID and then is transformed into `parent` in the function. Also you should probably use [Sprintf](https://golang.org/pkg/fmt/#Sprintf)  instead of concatentation
3. The idea of `parent` is new to me in the Google API and I don't think I have seen examples in our codebase but I do see [some context is provided in the docs](https://godoc.org/google.golang.org/api/serviceusage/v1#NewServicesService) I would leave a brief comment that would provide some context of why we need to prepend "projects/" before projectID